websocket/2.1/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Jakarta WebSocket 2.1 (under development)"
+date: 2021-04-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta WebSocket defines a API for Server and Client Endpoints
+for the WebSocket protocol (RFC6455).
+
+* [Jakarta WebSocket 2.1 Release Record](https://projects.eclipse.org/projects/ee4j.websocket/releases/2.1)
+  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
```

Comment:
I would drop this line.  Or, if you want to make an educated guess on where the Jakarta EE 10 Release Plan will be...  ;-)